tests/IResearch/IResearchLinkCoordinator-test.cpp

```diff
@@ -294,7 +294,7 @@ TEST_F(IResearchLinkCoordinatorTest, test_create_drop) {
         "\"shards\":{} }");
 
     EXPECT_TRUE((ci->createCollectionCoordinator(vocbase->name(), collectionId, 0, 1,
-                                                 false, collectionJson->slice(), 0.0)
+                                                 1, false, collectionJson->slice(), 0.0)
```

Comment:
C++ really needs named parameters. This is impossible to reason about.

Reply:
agreed...